src/rantanplan/core.py

```diff
@@ -63,23 +1336,55 @@ def have_prosodic_liaison(first_syllable, second_syllable):
 """
 
 
-def hyphenate(word):
+def apply_exception_rules(word):
+    # 'Abrogar' and derivatives
+    if ABROG_RE.match(word):
+        match = ABROG_RE.search(word)
+        if match is not None:
+            word = match.group(1) + '-' + match.group(2)
```

Comment:
Might be slightly more Pythonic to write `word = match.group(1), match.group(2)` for all cases, and then, at the end, `return "-".join(word)`. Or even better, why not simply `word = match.groups()`?

Also, `.search()` will stop after the first match is found. Is that what we generally want?

Reply:
So, like this?
```
    match = ABROG_RE.search(word)
    if match is not None:
        word = match.groups()
    return "-".join(word)
```

Reply:
And about using search, the regex states that the pattern has to be at the beginning of the word so, maybe irrelevant?